tests/Examples/openfhe/naive_matmul.mlir

```diff
@@ -9,7 +9,7 @@
 !rlwe_pt = !lwe.rlwe_plaintext<encoding = #lwe.polynomial_evaluation_encoding<cleartext_start = 32, cleartext_bitwidth = 32>, ring = #ring_Z463187969_i32_1_x1024_, underlying_type = f32>
 !rlwe_ct = !lwe.rlwe_ciphertext<encoding = #lwe.polynomial_evaluation_encoding<cleartext_start = 32, cleartext_bitwidth = 32>, rlwe_params = <ring = #ring_Z463187969_i32_1_x1024_>, underlying_type = f32>
 module {
-  func.func @matmul(%arg0: tensor<1x16x!rlwe_ct>, %arg1: tensor<1x16x!rlwe_ct> {secret.secret}) -> tensor<1x16x!rlwe_ct> {
+  func.func @matmul(%arg0: tensor<1x16x!rlwe_ct>, %arg1: tensor<1x16x!rlwe_ct>) -> tensor<1x16x!rlwe_ct> {
```

Comment:
hah, thank you! my bad

Reply:
Actually, I think I might have added that in my last secretize-cleanup PR 🙈